.eslintrc

```diff
@@ -10,6 +10,7 @@
     "sourceType": "module",
     "parser": "@typescript-eslint/parser"
   },
+  "ignorePatterns": ["codex-ui/dist"],
```

Comment:
your branch should not contain this change 🤔
seems like you did smth wrong while merging branches